plugins/traction_innkeeper/pyproject.toml

```diff
@@ -14,6 +14,7 @@ python-dateutil = "^2.8.2"
 bcrypt = "^4.0.1"
 mergedeep = "^1.3.4"
 typing-extensions = "4.8.0"
+anoncreds = "^0.2.0"
```

Comment:
Duplicate, likely from merge (probably what's causing build failure)

Reply:
If it fails, I'll rerun to update the lock file or revert